examples/azure-rules-perf-optimized.yaml

```diff
@@ -20,6 +20,23 @@ groups:
           Install a SUSE supported OS release
         scored: true
       - id: 2.1.2
+        description: "Check os release"
+        audit: 'cat /etc/os-release'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP1"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP2"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP3"'
+        remediation: |
+          ## Abstract
+          You need at least SLES for SAP 15.1
+          ## Remediation
+          Install at least SLES for SAP 15.1
+          ## Reference
+          T.B.A.
```

Comment:
@lee-martin could you point us at where we document these version constraints?

Reply:
@stefanotorresi 

I have several points to raise here:

1.  While external contributes cannot see our Jira, https://jira.suse.com/browse/CFSA-83 does have the references for both of these checks (min OS version and HanaSR version) - admittedly I only added those details on May 3rd.
Anyway, please refer to section 5.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/   )

2. The check is "at least SLES for SAP 15.1" ,  but as far as I understand the implementation it only looks for SP1, SP2 and SP3. What about versions after that, SP4? Next major version perhaps?
Wouldn't it be easier just to check "VERSION_ID" gte 15.1 ?

3. Since SLES4SAP is a SLES at heart, it uses the SLES PRETTY_NAME, e.g. PRETTY_NAME="SUSE Linux Enterprise Server 15 SP1".  Which test systems were giving you a PRETTY_NAME with the SAP Applications names? That would be a bug.

4. In the Abstract we should use the full product name.
Could be good if the abstract were to quote the source reference, in this case it's a simple sentence.

5. Can we add a date to when the references was checked. As Angela pointed out recently thing change over time, so we should record when the reference was last checked.

6. I notice the test IDs just changed due to this PR. Is there some way we could keep test id's static so that we can refer to them consistently in discussions.

Reply:
> 5\. Can we add a date to when the references was checked. As Angela pointed out recently thing change over time, so we should record when the reference was last checked.

This is not an aspect specific for this particular check, so I would avoid addressing this right now.
Besides, since these rule sets are living documents within a version control system, I think trying to keep track of the date of the reference would be just a lot of overhead.

Reply:
> 6\. I notice the test IDs just changed due to this PR. Is there some way we could keep test id's static so that we can refer to them consistently in discussions.

As per our face-to-face discussion, this is bound to happen for the time being, and I don't think we can address this right now.